### PR TITLE
Update to Dokka 1.4.30

### PR DESCRIPTION
**Pull request by ricknout**

### CODE REVIEW

**Comment by ricknout** [comment]
Currently appears to be running into this issue when running `./gradlew clean dokkaHtmlMultiModule` locally
https://github.com/vanniktech/gradle-maven-publish-plugin/issues/208

File: generate_docs.sh [hunk 1]
```diff
@@ -7,4 +7,4 @@ sed -i 's/CONTRIBUTING.md/\/contributing/' docs/index.md
 sed -i 's/docs\/using-snapshot-version.md/using-snapshot-version/' docs/index.md
 
 # Build the docs
-./gradlew clean dokka
+./gradlew clean dokkaHtmlMultiModule
```
Comment (chrisbanes) [inline generate_docs.sh:10]: You don't need multi module here, but it doesn't hurt it use it. Also you've switched from gfm to html docs here, so you might need to tweak the publish script

**Review by chrisbanes — APPROVED** [review]

File: generate_docs.sh (hunk 1 above)
Comment (ricknout) [reply generate_docs.sh:10]: Ah yes, will change to `dokkaGfmMultiModule`

**Comment by ricknout** [comment]
This was the initial issue with Dokka `0.10.1`:
https://github.com/Kotlin/dokka/issues/1834

**Pull request merged**